src/rust/engine/process_executor/src/main.rs

```diff
@@ -170,6 +170,16 @@ struct Opt {
   #[structopt(long)]
   cas_root_ca_cert_file: Option<PathBuf>,
 
+  /// Path to file containing client certificates for the CAS server.
+  /// If not set, TLS will not be used when connecting to the CAS server.
+  #[structopt(long)]
+  cas_mtls_certs_file: Option<PathBuf>,
+
+  /// Path to file containing client key for the CAS server.
+  /// If not set, TLS will not be used when connecting to the CAS server.
```

Comment:
Similar comment about the wording as earlier.

Reply:
TLS -> mTLS.